woocommerce/woocommerce
# Add plugin installation request track for core profiler

### Changes proposed in this Pull Request:

Closes #39484.

This PR adds the track `wcadmin_coreprofiler_store_extensions_continue` with props `shown`, `selected`, and `unselected` when a user clicks on continue in the plugins step in core profiler.

- `shown` are plugins that are shown to the user whether it's installed & activated or not
- `selected` are plugins that are selected
- `unselected` are plugins that are unselected

### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. In a fresh site, open your browser developer console and run `localStorage.setItem( 'debug', 'wc-admin:*' );` to set debug for tracks.
2. Ensure no the following plugins are not already installed & activated: Jetpack, Pinterest, MailPoet, Google Listing & Ads
3. Go to `Onboarding wizard > Business Info` step (`/wp-admin/admin.php?page=wc-admin&path=%2Fsetup-wizard&step=business-info`)
4. Choose `Afghanistan` as the store country to have limited plugins
3. Click on `Continue`
5. Deselect Jetpack and MailPoet and click on `Continue`
6. Observe that the track debug is shown along with its props <br><img width="647" alt="image" src="https://github.com/woocommerce/woocommerce/assets/3747241/bb903580-f033-467e-a5e2-2ed1d660e3e5">
7. Wait until the installation finishes and press the back button to go back to the plugins step
8. Leave everything checked and click on `Continue`
9. Observe that the track debug is shown along with its props <br><img width="642" alt="image" src="https://github.com/woocommerce/woocommerce/assets/3747241/aa8017b2-bc54-4ce2-84bf-dc41d1f747f6">




### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
